android/app/src/main/java/org/fossasia/openevent/activities/SearchActivity.java

```diff
@@ -100,16 +88,27 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
     @Override
     public boolean onQueryTextChange(String query) {
-        if (query != null) {
-            searchQuery(query);
+        String query1 = query.toLowerCase(Locale.getDefault());
```

Comment:
No need for this temporary varialble, why not just add it inside the String.format call

Reply:
@srv-twry Sure